src/puppetlabs/trapperkeeper/services/webserver/jetty9_config.clj

```diff
@@ -90,38 +90,39 @@
    (schema/optional-key :follow-links) schema/Bool})
 
 (def WebserverRawConfig
```

Comment:
This is a lot of whitespace change. Could this be pared down to just the new key?